app/policies/division_policy.rb

```diff
@@ -18,4 +18,30 @@ def update?
   def destroy?
     division_admin(division: @record)
   end
+
+  def select?
+    show?
+  end
+
+  class Scope < Scope
```

Comment:
We really should be testing this as so much depends on it. There are some good docs online for testing scopes in Pundit. Inside UserPolicySpec I would put two describe blocks, one for permissions, and one for scope. Create a few different divisions (child, parent, etc.) and users with different roles (admin, member, none) and ensure that the scope resolves to the right divisions. Shouldn't take long.